README.rst

```diff
@@ -43,6 +43,18 @@ Installing
 
 Installing on Windows is not supported because we use symlinks inside Python packages.
 
+While ansible-lint project does not directly ships a container, the
```

Comment:
Either remove the word project or add the word the before ansible-lint.
ie, 'the ansible-lint project' or 'ansible lint does'

Change 'ships' to 'ship'